roles/pulp_common/tasks/main.yml

```diff
@@ -109,7 +109,11 @@
   set_fact:
     default_bin_path: "{{ systemd_show_env_path.stdout }}"
 
-- import_tasks: repos.yml
+- name: Enable epel repos
+  import_role:
+    name: pulp_repos
+  when: pulp_repos_enabled
+
```

Comment:
As we discussed, we could probably make this work better by doing role dependencies with variables (when pulp_redis or pulp_database doesn't need a repo enabled, we'd set an aforementioned set-of-3 "enable" variable to fals, or set the a set-of-3 repo-name-list variable to an empty list:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#using-role-dependencies